src/Extensions/Hosting.Services/Internal/OmexStatelessServiceRunner.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+using Microsoft.ServiceFabric.Services.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+
+
+	/// <summary>
+	/// Class to manage stateless service fabric service startup
+	/// </summary>
+	internal sealed class OmexStatelessServiceRunner : IOmexServiceRunner
+	{
+		private readonly string m_applicationName;
+		private readonly ServiceContextAccessor<StatelessServiceContext> m_contextAccessor;
+		private IEnumerable<IListenerBuilder<StatelessServiceContext>> ListenerBuilders { get; }
```

Comment:
Same comment about the getters

Reply:
Same answer :) https://github.com/microsoft/Omex/pull/153#discussion_r389413931